RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
+  is only available on images where it was already used for editing.
+
+- A new module 'tone equalizer', designed to replace the 'tone mapping' module,
+  brings an easy and safe way to adjust the tones in different areas of the
+  image. The module comes with 9 controls (available as fixed sliders or nodes
+  on a spline) covering the range from blacks to specular highlights. It also
+  features an advanced picker tool which can be moved over the image to make
+  tonal modifications to specific areas.
 
 ## New Features And Changes
 
 - A new module for handling 3D Lut transformations (PNG Hald-CLUT and
   Cube files are supported).
 
-- Many changes to denoise profiled. The variance stabilization transform
-  has changed, allowing a finer control of denoising. In particular,
-  it is possible to better denoise the shadows, and correct color casts
-  that may occur in the shadows. This change mainly improves the denoising
-  of high ISO images.
-  The non-local means mode has 2 new options: an option to scatter
-  the search zone, and an option to control the amount of details to keep.
-  In addition, one "auto" mode was added both for wavelets and non-local
-  means. This mode infers some parameters from the profile used.
-  The default values of the sliders are also changed dynamically in the same
-  way. As such, user who want a one click denoising can activate the module
-  and get straight away a good trade-off between noise smoothing and detail
-  preservation.
-  The auto modes allow the user to create their own presets that will
-  adapt to various ISOs.
-  Finally denoise profiled controls now have soft boundaries, which means
-  that users can enter values out of visible bounds using the keyboard if
-  they need to.
-
-- A new profile ‘histogram profile’ has been added on the same pop-up
-  that the softproof one on the darkroom. It controls the color space
-  of the histogram, color picker and overexposed check. When gamut or
-  softproof checks are active the histogram and color picker use the
-  softproof profile, otherwise they use the histogram profile. The
-  overexposed check always use the histogram profile.
-
-- A new profile 'work profile' has been added to the input color profile
-  module. Now RGB modules between the input and output color profile modules
-  will work with this profile.
-
-- A new color picker has been added to the blend module next to the
-  existing one that adjusts the range sliders based on the selected
-  area from the image. Click to adjust the input image slider,
-  <kbd>Ctrl+click</kbd> to adjust the out image one.
-
-- The picasa module has been renamed to googlephoto and completely
-  rewrote to support the new Google Photo API. It is now again
+- Many improvements to the 'denoise (profiled)' module. The degree of shadow
+  denoising can be can be controlled, including the correction of color casts
+  (mainly improves high-ISO images). An "auto" mode was added which infers
```

Comment:
What is auto mode? don't see it in the module.

Reply:
In the "mode" drop down menu, there are two which are "auto".  I can probably word that part a little more clearly...

Reply:
I was aware but had my new user hat on ;)